### DISCUSSION TITLE
How to return custom 404 error?

### DISCUSSION THREAD

**Discussion by imadatyatalah** [prisma/prisma#8458] [Q&A] [answered]
I tried to configure `rejectOnNotFound` [globally](https://www.prisma.io/docs/reference/api-reference/prisma-client-reference#throw-a-custom-error-per-model-and-operation-if-record-is-not-found) but it returns just `{ clientVersion: '2.27.0' }`

**Comment by ryands17** [comment, marked as answer]
@imadatyatalah 👋 

You can get the message of the error as follows:

```ts
try {
 // prisma call
} catch(e) {
  console.log(e.message)
}
```

This will give you the correct message with regards to the model. I would still suggest creating an issue here to include the error code for `rejectOnNotFound` errors.

**Reply by ryands17** [reply]
@imadatyatalah I hope that answers your question 🙂

**Reply by imadatyatalah** [reply]
Yeah it solves my problem, Sorry for the late response!
Thanks @ryands17

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that have already been marked as answered but remain open.

If this discussion still requires further input or clarification, feel free to reopen it or start a new one with updated details. Your contributions are invaluable to the community, and we’re here to help!

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and ongoing support of the Prisma community!
